ownCloud/Client/Actions/ClientItemResolvingCell.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ClientItemResolvingCell.swift
+//  ownCloud
+//
+//  Created by Felix Schwarz on 18.07.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2019, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+
+class ClientItemResolvingCell: ClientItemCell {
+	var itemTracker : OCCoreItemTracking?
+
+	// MARK: - Resolve item from path
+	var itemResolutionPath : String? {
+		didSet {
+			self.item = nil
+
+			if let atPath = itemResolutionPath {
+				self.itemResolutionLocalID = nil
+				self.itemTracker = core?.trackItem(atPath: atPath, trackingHandler: { (error, item, isInitial) in
+					if error == nil, let item = item, isInitial {
+						OnMainThread {
+							self.item = item
+							self.itemTracker = nil // unless isInitial is removed from above "if", that's it - we can stop tracking as any updates won't be used anyway
+						}
+					} else if (error != nil) || (item == nil) {
+						OnMainThread {
+							self.resolutionFailed(error: error)
```

Comment:
This function does not contain code. Remove this line and the function or should this function have any code inside?

Reply:
It's just a subclassing point for now, with no default functionality.

In case we have an item cell that's resolving its item from a path in the future (by subclassing ClientItemResolvingCell) and we want to handle the case that it can't be resolved, that'd be where that case would be handled.